osidb/tests/test_rls.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+from django.conf import settings
+from django.db import transaction
+from django.db.utils import ProgrammingError
+
+from osidb.core import set_user_acls
+from osidb.models import Flaw
+from osidb.tests.factories import FlawFactory
+
+pytestmark = pytest.mark.enable_rls
+
+
+class TestRLS:
+    # XXX: The following tests use Flaw for a model to test ACLs on,
```

Comment:
Leftover `XXX`?

Reply:
intended, it's just documentation

Reply:
Not sure whether it is a convention but at least my editor (Vim) highlights **XXX** string by default (the same ways is it does for **TODO**).

Reply:
Yeah, it is used very often and it means sth like `TODO`/`FIX`/`NOTE`.